plugins/kubernetes/app/models/watchers/deploy_watcher.rb

```diff
@@ -10,6 +10,7 @@ class DeployWatcher
     def initialize(release)
       @release = release
       @current_rcs = {}
+      @pod_timer = after(ENV.fetch('KUBERNETES_POD_TIMEOUT', 600)) { pod_timeout }
```

Comment:
This is 10 mins right? Should we increase the default to last a little longer? I'm just thinking that 10 mins might not be too long for a medium-sized deploy. 

Hum, nvm, I just noticed we are updating the timer each time we get a valid update for a pod.